tests/test_parastell.py

```diff
@@ -29,28 +29,171 @@ def remove_files():
             Path.unlink(file)
 
 
+def check_cubit():
+    try:
+        import cubit
+
+        return True
+    except ImportError:
+        return False
+
+
 def check_surfaces_and_volumes(filename, num_surfaces_exp, num_volumes_exp):
     dagmc_model = dagmc.DAGModel(str(Path(filename).with_suffix(".h5m")))
 
     assert len(dagmc_model.surfaces) == num_surfaces_exp
     assert len(dagmc_model.volumes) == num_volumes_exp
 
 
+def create_ivb_cad_magnets_from_filaments(stellarator_obj):
```

Comment:
Maybe we should have docstrings or other comments to clarify what each test should actually be testing and what result is being used to accomplish the test